src/3p-frame.js

```diff
@@ -75,7 +72,7 @@ function getFrameAttributes(parentWindow, element, opt_type) {
     referrer: viewer.getUnconfirmedReferrerUrl(),
     canonicalUrl: docInfo.canonicalUrl,
     pageViewId: docInfo.pageViewId,
-    clientId: element.getAttribute('ampcid'),
+    clientId: null,
```

Comment:
I checked facebook & twitter example page, all good.
One minor behavior change: For 3p iframe that is NOT created by amp-ad, `clientId` will be `undefined` instead of `null` after this PR. I checked the behavior is not documented before, so we should be good with this change.
Delete this line of code.